src/averageAges.js

```diff
@@ -54,9 +66,29 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
+function calculateAvg(peopleArray) {
```

Comment:
Check @dinaminahi comment.
Use more descriptive names. 
Try to describe action inside this function.
`calculateAverageAges` or `getAverageAges` would be better